source/features/global-search-filters.tsx

```diff
@@ -1,54 +0,0 @@
-import React from 'dom-chef';
-import {XIcon} from '@primer/octicons-react';
-import elementReady from 'element-ready';
-import * as pageDetect from 'github-url-detection';
-
-import features from '../feature-manager';
-import SearchQuery from '../github-helpers/search-query';
-import {getUsername} from '../github-helpers';
-
-async function init(): Promise<void> {
-	const filters = [
-		['Forks', 'fork:true'],
-		['Private', 'is:private'],
-		['Yours', 'user:' + getUsername()!],
-		['Authored', 'author:@me'],
```

Comment:
Bottom left sidebar, but no `@me` automomplete

<img width="266" alt="Screenshot" src="https://user-images.githubusercontent.com/1402241/215421383-6527e24d-3ff1-443f-b2a7-ec6f27b19934.png">


Arguably this is already available via the Issues/PRs global links in the header